neqo-transport/tests/network.rs

```diff
@@ -69,6 +70,28 @@ simulate!(
     ],
 );
 
+simulate!(
+    connect_tail_drop_jitter,
+    [
+        ConnectionNode::new_client(boxed![ReachState::new(State::Confirmed)]),
+        TailDrop::dsl_uplink(),
+        Delay::new(ZERO..JITTER),
```

Comment:
is this going to add reordering? I need o check...

Reply:
Yes, this adds reordering.  As you say, we probably do want a random delay without reordering (and probably a queue where small packets can jump the queue).  We can add that later.

This reordering already turned out to be useful in uncovering a bug.